module/EntraBeta/config/ModuleSettings.json

```diff
@@ -4,6 +4,7 @@
     "newPrefix" : "EntraBeta",
     "typePrefix" : "Microsoft.Open.",
     "destinationModuleName" : [
+      "Microsoft.Graph.Beta",
```

Comment:
Please consider removing this entry (which will install *all* Microsoft.Graph.Beta sub-modules) unless truly necessary.